### PR TITLE
Passing parameters as an array

### PR DESCRIPTION
**Pull request by ebashu-on-holidays**
Now it will work:
```yaml
cssClass: row
fields:
    price_section:
        label: Cost of goods
        type: section
        span: storm
        cssClass: col-xs-12
```

Just like this:
```yaml
tabs:
    cssClass: row
    fields:
        price_section:
            label: Cost of goods
            type: section
            span: storm
            cssClass: col-xs-12
```

I have previously received an error: Cannot use object of type stdClass as array
https://github.com/octobercms/october/blob/master/modules/backend/classes/FormTabs.php#L94

### CODE REVIEW

File: modules/backend/widgets/Form.php [hunk 1]
```diff
@@ -431,7 +431,7 @@ protected function defineFormFields()
             $this->fields = [];
         }
 
-        $this->allTabs->outside = new FormTabs(FormTabs::SECTION_OUTSIDE, $this->config);
+        $this->allTabs->outside = new FormTabs(FormTabs::SECTION_OUTSIDE, (array)$this->config);
```
Comment (LukeTowers) [inline modules/backend/widgets/Form.php]: Put a space between `(array)` and `$this` and I'll merge it

**Comment by ebashu-on-holidays** [comment]
@LukeTowers , all is ready

**Comment by LukeTowers** [comment]
Perfect, thanks @ebashu-on-holidays!

**Comment by LukeTowers** [comment]
This pull request has been made to the wrong branch. Please review [the contributing guidelines](https://github.com/octobercms/october/blob/master/CONTRIBUTING.md#pull-requests) as all PRs need to be made to the develop branch.

I'll fix it for you this time, but please ensure you make any future PRs to the develop branch, not the master branch.

**Pull request merged**